diff --git a/www_data/lib/mail.php b/www_data/lib/mail.php
index fc2246e..2e060e3 100644
--- a/www_data/lib/mail.php
+++ b/www_data/lib/mail.php
@@ -15,10 +15,10 @@ function mail_init(){
 
        $mail->isSMTP();
        $mail->Host       = SMTP_HOST;
-       $mail->SMTPAuth   = true;
+       $mail->SMTPAuth   = false;
        $mail->Username   = SMTP_MAIL;
        $mail->Password   = SMTP_PASSWORD;
-       $mail->SMTPSecure = PHPMailer::ENCRYPTION_SMTPS;
+       $mail->SMTPSecure = false;
        $mail->Port       = SMTP_PORT;
        $mail->CharSet    = 'UTF-8';
 