diff --git a/www_data/lib/mail.php b/www_data/lib/mail.php
index 9b58b50..ff99f07 100644
--- a/www_data/lib/mail.php
+++ b/www_data/lib/mail.php
@@ -15,13 +15,13 @@ function mail_init(){
 
        $mail->isSMTP();
        $mail->Host       = SMTP_HOST;
-       $mail->SMTPAuth   = true;
+       $mail->SMTPAuth   = false;
        $mail->Username   = SMTP_MAIL;
        $mail->Password   = SMTP_PASSWORD;
-       $mail->SMTPSecure = PHPMailer::ENCRYPTION_SMTPS;
+       $mail->SMTPSecure = false;
        $mail->Port       = SMTP_PORT;
        $mail->CharSet    = 'UTF-8';
-
+       
        $mail->setFrom(SMTP_MAIL, 'moncycle.app');
        return $mail;
 }
@@ -105,5 +105,3 @@ function mail_body_relance ($nom, $mail1) {
        </div>
        HTML;
 }
-
-