diff --git a/www_data/lib/mail.php b/www_data/lib/mail.php
index 2a60306..a6dc283 100644
--- a/www_data/lib/mail.php
+++ b/www_data/lib/mail.php
@@ -15,7 +15,7 @@ function mail_init(){
 
        $mail->isSMTP();
        $mail->Host       = SMTP_HOST;
-       $mail->SMTPAuth   = true;
+       $mail->SMTPAuth   = false;
        $mail->Username   = SMTP_MAIL;
        $mail->Password   = SMTP_PASSWORD;
        $mail->SMTPSecure = PHPMailer::ENCRYPTION_SMTPS;